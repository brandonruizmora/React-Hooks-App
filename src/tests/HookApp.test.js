import '../setupTest';
import { shallow } from "enzyme";
import { HookApp } from "../HookApp";

describe('Pruebas en <HookApp />', () => {
    test('Debe renderizar correctamente', () => {
        const wrapper = shallow(<HookApp />);
        expect(wrapper).toMatchSnapshot();
    });
    
});
