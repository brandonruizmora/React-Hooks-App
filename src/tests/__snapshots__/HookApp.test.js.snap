// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HookApp /> Debe renderizar correctamente 1`] = `
<div>
  Hola mundo
</div>
`;
